task/01-strings-tasks.js

```diff
@@ -114,7 +114,13 @@ function removeLeadingAndTrailingWhitespaces(value) {
  *   'cat', 3 => 'catcatcat'
  */
 function repeatString(value, count) {
-    throw new Error('Not implemented');
+    let str = '';
+
+    for (let i = 0; i < count; i += 1) {
+        str += value;
+    }
+
```

Comment:
JS has the function
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat

Reply:
fix